bouncer/bouncer.py

```diff
@@ -0,0 +1,53 @@
+
+import json
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+SECRETS_MANAGER_URL = "https://secretsmanager.us-west-2.amazonaws.com"
+SECRET_KEY = 'email'
+
+
+class SecretManagerException(Exception):
+    pass
+
+
+class Bouncer(object):
+    """
+    Client convenience class to perform email whitelist authorization
+    """
+    session = boto3.session.Session()
+    client = session.client(service_name='secretsmanager', endpoint_url=SECRETS_MANAGER_URL)
+
+    def __init__(self, secret_name: str):
+        """
+        :param secret_name: The name of the secret in
+        the AWS Secret Manager. For example: commons/dev/whitelist
+        """
+        self.secret_name = secret_name
+
+    def is_authorized(self, email: str) -> bool:
+        """
+        Identify if the given email address is included in the email whitelist.
+
+        :param email: Google email address for which authorization is to be determined
+        :return: True if the email address is authorized, False if not authorized
+        :exception: <TBD> If an error occurs.
```

Comment:
Please replace `<TBD>` with `SecretManagerException`

Please also add any other specific exceptions you are aware of that may occur, such as a KeyError if the "email" key is not defined in the secret.

Reply:
Thanks! I had overlooked this.